lib/generators/rubyblok/migration_generator.rb

```diff
@@ -0,0 +1,28 @@
+require "rails/generators"
+require "rails/generators/active_record"
+
+module Rubyblok
+  module Generators
+    class MigrationGenerator < ::Rails::Generators::NamedBase
+      include ::Rails::Generators::Migration
+      source_root File.expand_path("../templates", __dir__)
+      desc "Installs rubyblok migration and model files."
+
+      def install
+        migration_template("migration.rb.erb", "db/migrate/create_rubyblok_#{plural_file_name}.rb",
```

Comment:
we should distinguish two cases:
1. the model already exists: in this case we should not create the model just include the rubylok modul, also we shouldn't create the table just add the new fields
2. the model already does not exist: in this case the current implementation is fine